### ISSUE TITLE
Correct resampling of multiple swaths that do not fit into memory

### ISSUE THREAD

**Issue by cpaulik** [pytroll/pyresample#54]
We are currently working on the problem of resampling multiple swaths in a correct manner. Our main concern are edge effects when stitching swaths or parts of orbits together. We mostly use a hamming window for resampling so we do not want to ignore the edges of our swaths. We do have a working solution for the problem and I am currently looking into the best way to open source it.

My main questions are:

1. Would you consider this functionality a part of pyresample or should wrappers around pyresample manage inter-swath resampling.
2. Are you aware of other projects that tackle the same issue which we could contact if your answer to question 1 is no. 

Otherwise we will open source it as our own project and keep you informed if you are interested.

**Comment by mraspaud** [comment]
@cpaulik Sorry for the late answer. 
I would say it would be a nice addition to pyresample! Do you have some examples ?

**Comment by floriankrb** [comment]
I have the same use case of resampling swaths with high memory consumption.
 
I saw some dependencies on dask in pyresample. Would using dask/xarray be a solution to avoid loading everything in memory ?

@cpaulik I would be very interested in knowing more about your working solution.

**Comment by mraspaud** [comment]
Yes, we're working with dask and xarray atm, and that helps with memory consumption. Feel free to join us on slack to discuss it, as it's still quite experimental (https://pytrollslackin.herokuapp.com/)

**Comment by mraspaud** [comment]
@cpaulik I'm assigning you to this issue, feel free to contact us if you need help with a PR

**Comment by mraspaud** [comment]
No rush, we can wait :)

About 2. I'm wondering how that data structure would be implemented. An adjacency graph/matrix of sorts maybe ?
